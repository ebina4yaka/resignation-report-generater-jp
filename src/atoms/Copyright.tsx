import { ReactElement } from 'react'
import Typography from '@material-ui/core/Typography'
import MuiLink from '@material-ui/core/Link'

export default function Copyright(): ReactElement {
  return (
    <Typography variant="body2" color="textSecondary" align="center">
      {'Copyright Â© '}
      <MuiLink color="inherit" href="https://github.com/ebina4yaka">
        ebina4yaka
      </MuiLink>
      {` 2020.`}
    </Typography>
  )
}
