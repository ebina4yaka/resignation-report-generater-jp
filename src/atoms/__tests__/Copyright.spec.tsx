import renderer from 'react-test-renderer'
import Copyright from '../Copyright'

test('Snapshot test for Copyright', () => {
  const tree = renderer.create(<Copyright />).toJSON()
  expect(tree).toMatchSnapshot()
})
