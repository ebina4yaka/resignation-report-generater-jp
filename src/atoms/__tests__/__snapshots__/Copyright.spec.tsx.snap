// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Copyright 1`] = `
<p
  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
>
  Copyright Â© 
  <a
    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
    href="https://github.com/ebina4yaka"
    onBlur={[Function]}
    onFocus={[Function]}
  >
    ebina4yaka
  </a>
   2020.
</p>
`;
